src/components/GoodsList.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { GoodItem } from './GoodItem';
+
+interface GoodsListProps {
```

Comment:
I prefer to name it just Props(it is easier to always work with the same name), but your approach is also good, especially if you need to export this interface